concept_data/metric.py

```diff
@@ -7,4 +7,5 @@
     ('person_birth_date', None),
     ('postcode_of_usual_address', None),
     ('nhs_number', None),
+    ('ethnic_category', None)
```

Comment:
the test urgency (i think i prefer this name over priority) metric needs to be added in here also, so...
```suggestion
    ('ethnic_category', None),
    ('test_urgency' None),
```